pkg/sdk/v1beta1/kafkacluster_types.go

```diff
@@ -156,6 +156,8 @@ type BrokerConfig struct {
 	BrokerIngressMapping []string `json:"brokerIngressMapping,omitempty"`
 	// InitContainers add extra initContainers to the Kafka broker pod
 	InitContainers []corev1.Container `json:"initContainers,omitempty"`
+	// ResourceRequirements for initContainers
+	InitContainersResourceRequirements *corev1.ResourceRequirements `json:"initContainersResourceRequirements,omitempty"`
```

Comment:
This field is redundant. The `InitContainers []corev1.Container` field already defines resource requirements for each init container.